### PR TITLE
Moved flutter_lints to dev_dependencies

### PR DESCRIPTION
**Pull request by smihica**
Hi!

I thought it was a mistake...?

It is annoying because it fixes the version of the user's flutter_lints to 2.0.1 because of dependency conflicts.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -10,10 +10,10 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_lints: ^2.0.1
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  flutter_lints: ^2.0.1
```
Comment (smihica) [inline pubspec.yaml:17]: This is it! :)

Comment (idootop) [reply pubspec.yaml:17]: LGTM!

**Comment by idootop** [comment]
Thanks, bro! This repository has been deprecated, and I highly recommend migrating to [nested_scroll_view_plus](https://github.com/idootop/nested_scroll_view_plus).

**Comment by smihica** [comment]
Oh nice! I'll move it later.

Thank you! 😀

**Pull request merged**